source/docs/use/manage/type/dashboard.md

```diff
@@ -32,10 +33,6 @@ HACS has a custom path for serving up dashboard elements `/hacsfiles/`, also ref
 
 ??? nerd-mode
 
-    - Every dashboard you have downloaded with HACS has an **Open source** option on the hamburger menu. This is useful when troubleshooting:
-        - If you can see a code wall (the underlying `.js` file), everything is OK.
-        - But if you get a 404, try downloading it again.
-    - `.gz` example: CCH (Compact Custom Header) is a fairly popular element for Lovelace. 
-        - When you use `/local`, you will transfer the `.js` file, which is 101&nbsp;kB. 
-        - However, if you use HACS and reference the dashboard with `/hacsfiles`, it will serve the 20&nbsp;kB `.gz` version automatically. (Sizes are accurate for version 1.4.7 of CCH).
+    - Every dashboard you have downloaded with HACS has an **Open source** option on the hamburger menu. This is useful when troubleshooting; if you can see a code wall (the underlying `.js` file) everything is OK, but if you get a 404, try downloading it again.
+    - `.gz` example: CCH (Compact Custom Header) is a fairly popular element for Lovelace. When you use `/local`, you will transfer the `.js` file, which is 101&nbsp;kB. However, if you use HACS and reference the dashboard with `/hacsfiles`, it will serve the 20&nbsp;kB `.gz` version automatically (sizes accurate for version 1.4.7 of CCH).

```

Comment:
```suggestion
    - Every dashboard you have downloaded with HACS has an **Open source** option on the hamburger menu. This is useful when troubleshooting:
        - If you can see a code wall (the underlying `.js` file), everything is OK.
        - But if you get a 404, try downloading it again.
    - `.gz` example: CCH (Compact Custom Header) is a fairly popular element for Lovelace. 
        - When you use `/local`, you will transfer the `.js` file, which is 101&nbsp;kB. 
        - However, if you use HACS and reference the dashboard with `/hacsfiles`, it will serve the 20&nbsp;kB `.gz` version automatically. (Sizes are accurate for version 1.4.7 of CCH).
```